### PR TITLE
added examples folder and .py files

### PR DESCRIPTION
**Pull request by mrmcwethy**
#https://github.com/adafruit/circuitpython/issues/493

added examples files for acceleration, pixels, shake, tapdetect, tone, and touched.

### CODE REVIEW

**Comment by kattni** [comment]
We may want to consider also including a simpler tap detection example. The purpose of that example was to show how single and double tap can be used together, it wasn't meant to be the typical usage of it. I think it's fine to include it, I simply think we should have a more likely use-case.

**Comment by mrmcwethy** [comment]
@kattni i can do that.

File: examples/acceleration.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from adafruit_circuitplayground.express import cpx
```
Comment (kattni) [inline examples/acceleration.py]: I would add an `import time` and `time.sleep(0.1)` at least into this one so it is more reasonable to read.

Comment (mrmcwethy) [reply examples/acceleration.py]: ok

Comment (mrmcwethy) [reply examples/acceleration.py]: ok

**Comment by kattni** [comment]
Looks great, thank you!

**Comment by mrmcwethy** [comment]
@kattni  Thanks for the review

**Pull request merged**